TOC-tidb-cloud.md

```diff
@@ -99,6 +99,7 @@
     - [Compatibility](/tiflash/tiflash-compatibility.md)
   - [Scale a TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md)
   - [Pause or Resume a TiDB Cluster](/tidb-cloud/pause-or-resume-tidb-cluster.md)
+  - [Configure Maintenance Window](/tidb-cloud/configure-maintenance-window.md)
```

Comment:
```suggestion
  - [Configure a Maintenance Window](/tidb-cloud/configure-maintenance-window.md)
```

Reply:
"Configure Maintenance Window" here means "Configure the Maintenance Window feature"